src/server/api/routes/orders.router.js

```diff
@@ -0,0 +1,50 @@
+/* TODO: This is just an example file to illustrate API routing and
+documentation. Can be deleted when the first real route is added. */
+
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const ordersController = require('../controllers/orders.controller');
+
+/**
+ * @swagger
+ * /orders?user=user_id:
+ *  get:
+ *    tags:
+ *    - Orders
+ *    summary: Get orders by user ID
+ *    description:
+ *      Will return all orders for specific user.
+ *    produces: application/json
+ *    parameters:
+ *     - in: query
+ *       name: user
+ *       schema:
+ *         type: integer
+ *         required: true
+ *         description: Get all orders of specific user with user_id
+ *
+ *    responses:
+ *      200:
+ *        description: Successful request
+ *      404:
+ *        description: incorrect entry with the provided id
+ *      5XX:
+ *        description: Unexpected error.
+ */
+router.get('/', (req, res, next) => {
+  ordersController
+    .getOrderByUserId(req.query.user)
+    .then((result) => {
+      if (result.length === 0) {
+        res.status(404).send('The user ID you provided does not exist');
```

Comment:
Could this also happen if the user does not have any orders?
How would we tell the difference?